static/app/components/onboarding/productSelection.tsx

```diff
@@ -391,18 +403,6 @@ export function ProductSelection({
           checked={urlProducts.includes(ProductSolution.PERFORMANCE_MONITORING)}
         />
       )}
-      {products.includes(ProductSolution.SESSION_REPLAY) && (
```

Comment:
this changes the order of the product selection buttons (see PR description why)
- from Tracing - Session Replay - Profiling
- to Tracing - Profiling - Session Replay

we might want to reconsider this, not a strong opinion of mine :)